sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -132,6 +138,7 @@
         "parameters": [
           {
             "name": "top",
+            "description": "Number of configurations to retrieve",
```

Comment:
Do you think the service may override this value? We could ask them about this later if you leave a TODO here. ADT had something like this API where the service may override if you specify a number too large

Reply:
I see. We can ask the service team about it.